core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java

```diff
@@ -134,4 +139,8 @@ public boolean isAsyncCommitEnabled() {
   public boolean isAsyncRollbackEnabled() {
     return asyncRollbackEnabled;
   }
+
+  public boolean isDebugging() {
```

Comment:
```suggestion
  public boolean isDebugEnabled() {
```
This might be better to make it consistent with other boolean configurations.

Reply:
I will rename this `isMetadataColumnsFilteringEnabled` and update the rest of the code in consequence.

Reply:
After our discussion above, I will rename it to `isIncludeMetadataEnabled()`